.circleci/config.yml

```diff
@@ -103,9 +103,10 @@ jobs:
               TEST_FILES=$(circleci tests glob "**/*_test.py")
               TEST_FILES=$(echo "$TEST_FILES" | sed -E 's/\S*demisto_sdk\/commands\/init\/templates\S*\.py//g')
               TEST_FILES=$(echo "$TEST_FILES" | sed -E 's/\S*demisto_sdk\/tests\/integration_tests\S*\.py//g')
-              TEST_FILES=$(echo "$TEST_FILES" | circleci tests split --split-by=timings)
+              TEST_FILES=$(echo "$TEST_FILES" | circleci tests split --split-by=timings )
 
               mkdir test-results
+              sudo npm install -g markdownlint-cli
```

Comment:
1. Please move this line into `package.json`, and have the CI image install it with `-g` (see `install_node_ci`)
2. Please make sure, before merging, that we install the same under the `content` repo